hera.rb

```diff
@@ -0,0 +1,69 @@
+class Hera < Formula
+  desc "Hera: Ewasm virtual machine conforming to the EVMC API"
+  homepage "https://github.com/ewasm/hera"
+
+  stable do
+    url "https://github.com/ewasm/hera/archive/v0.2.4.tar.gz"
+    sha256 "6d1bb30764c892f1aaada26c08add021a981d1d27638d2c9e198ca831544f9d1"
+
+    resource "cmake/cable" do
+      url "https://github.com/ethereum/cable.git", tag: "v0.2.17"
+    end
+
+    resource "evmc" do
+      url "https://github.com/ethereum/evmc.git", tag: "v6.2.2"
+    end
+  end
+
+  head "https://github.com/ewasm/hera.git"
+
+  option "with-debugging", "Build with debugging features and messages"
+  option "without-shared-libraries", "Build libraries as static"
+  option "with-binaryen", "Build with binaryen"
+  option "without-wabt", "Build without wabt"
+  option "with-wavm", "Build with WAVM"
```

Comment:
Got all your options here, @axic, although do let me know if you think I've missed anything. The typical pattern is to declare the opposite of the default as the option, thus the discrepancy between them (some are `with-`, some are `without-`.) I went with what is defaulted in the CMakeLists.txt file for reference.